scripts/mutect2_wdl/mutect2.wdl

```diff
@@ -576,12 +576,12 @@ task M2 {
         touch f1r2.tar.gz
         echo "" > normal_name.txt
 
-        gatk --java-options "-Xmx~{command_mem}m" GetSampleName -R ~{ref_fasta} -I ~{tumor_bam} -O tumor_name.txt -encode
-        tumor_command_line="-I ~{tumor_bam} -tumor `cat tumor_name.txt`"
+        gatk --java-options "-Xmx~{command_mem}m" GetSampleName -R ~{ref_fasta} -I ~{tumor_bam} --read-index ~{tumor_bai} -O tumor_name.txt -encode
+        tumor_command_line="-I ~{tumor_bam} --read-index ~{tumor_bai} -tumor `cat tumor_name.txt`"
```

Comment:
Is it worth updating our WDLs to use `--read-index` if that argument is just going to be replaced soon with bundle files? Or did you update this particular one just so that @davidbenjamin could try this new feature out?

Reply:
I was just updating this so that I could make sure mutect works, but I didn't actually build in a test with the wdl and the new files, I'll revert it and either David can do it when he wants to  or it can wait for the bundle files.